bin/alluxio

```diff
@@ -258,7 +258,12 @@ function main {
   ;;
   "runTest")
     CLASS="alluxio.cli.TestRunner"
-    PARAMETER="alluxio://${ALLUXIO_MASTER_HOSTNAME}:19998"
+    local master_hostname=$(${BIN}/alluxio getConf alluxio.master.hostname)
+    if [[ -z ${master_hostname} ]]; then
+      echo "alluxio.master.hostname or ALLUXIO_MASTER_HOSTNAME must be configured to use runTest" >&2
```

Comment:
would it make sense to do offer ALLUXIO_MASTER_HOSTNAME as an option since it will be deprecated?